Standards/scs-XXXX-wN-key-manager-implementation-testing.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "SCS Key Manager Standard: Implementation and Testing Notes"
+type: Supplement
+track: IaaS
+status: Proposal
+supplements:
+  - scs-XXXX-v1-vN-key-manager-standard.md
+---
+
+## Implementation
+
+A Key Manager service can have different backends, for Barbican these are called Plugins.
+The standard plugin is `simple_crypto`, which has the Master-KEK written in the Barbican config file.
+To secure that Master-KEK when the `simple_crypto` plugin is used it is advised to protect the Barbican config e.g. through running Barbican in an enclave.
```

Comment:
"to [A] when [B] it is advised to [C]" => "when [B] [A] can be achieved by [C]"

Reply:
I rephrased it.